api4/system.go

```diff
@@ -46,7 +46,8 @@ func (api *API) InitSystem() {
 	api.BaseRoutes.APIRoot.Handle("/database/recycle", api.APISessionRequired(databaseRecycle)).Methods("POST")
 	api.BaseRoutes.APIRoot.Handle("/caches/invalidate", api.APISessionRequired(invalidateCaches)).Methods("POST")
 
-	api.BaseRoutes.APIRoot.Handle("/logs", api.APISessionRequired(getLogs)).Methods("GET")
+	api.BaseRoutes.APIRoot.Handle("/logs", api.APISessionRequired(getLogs)).Methods("POST")
```

Comment:
This is changing the API interface, I would suggest instead of moving it to `/logs/old`, keep it as `/logs`, and create a new one that I would use something like `/logs/query`, that way you are not breaking the API backward compatibility and you are giving a more explicit name to the API that allows you to query the logs.

Reply:
Agreed, we cannot simply do this, changing method is a breaking change for the API.